source/deployment/ldap-group-sync.rst

```diff
@@ -1,30 +1,35 @@
 .. _ldap-group-sync:
 
-AD/LDAP Groups (Alpha) (E20)
-============================
-
-This feature is currently offered as an alpha release for Enterprise Edition E20 customers via a custom build. Please see `this forum post <https://forum.mattermost.org/t/ldap-group-sync-alpha-release/6351>`__ for more details. 
+AD/LDAP Groups (Experimental) (E20)
```

Comment:
Do we need to update the FAQ items on why LDAP group sync is in alpha/experimental, and when it will be in beta?